### PR TITLE
enh(Input): Reduce input sizes

### PR DESCRIPTION
**Pull request by bdauria**
This makes further ajustements in size for inputs (Select, Text, Autocomplete). This allows to have a more ’compact’ aspect, so that the Filter section doesn’t look disproportionate to the Listing in a Listing page. 

Before: 
Capture d’écran 2021-01-26 à 17 44 15

After:
Capture d’écran 2021-01-26 à 17 36 54

### CODE REVIEW

**Comment by jdelpierre** [comment]
the "Filters" and "Criteria" labels have been removed ?

File: src/InputField/Select/index.tsx [hunk 1]
```diff
@@ -84,6 +87,7 @@ const SelectField = ({
         inputProps={{
           'aria-label': ariaLabel,
           className: clsx({
+            [classes.input]: true,
             [classes.noLabelInput]: !label && !compact,
             [classes.compact]: compact,
           }),
```
Comment (Thebarda) [inline src/InputField/Select/index.tsx:93]:
```suggestion
          className: clsx(classes.input, {
            [classes.noLabelInput]: !label && !compact,
            [classes.compact]: compact,
          }),
```

File: src/InputField/Text/index.tsx [hunk 2]
```diff
@@ -89,6 +92,7 @@ const TextField = ({
       inputProps={{
         'aria-label': ariaLabel,
         className: clsx({
+          [classes.input]: true,
           [classes.noLabelInput]: !label && !compact,
           [classes.compact]: compact,
         }),
```
Comment (Thebarda) [inline src/InputField/Text/index.tsx:98]:
```suggestion
        className: clsx(classes.input, {
          [classes.noLabelInput]: !label && !compact,
          [classes.compact]: compact,
        }),
```

**Review by Thebarda — APPROVED** [review]

**Comment by bdauria** [comment]
> the "Filters" and "Criterias" labels have been removed ?

Yes, but this is on centreon’s side.

**Review by Thebarda — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**